src/cosmos/cosmosTypes.ts

```diff
@@ -164,3 +174,51 @@ export const asThornodeNetwork = asObject({
   // total_reserve: asString, // '8596638591565880',
   // vaults_migrating: false
 })
+
+/**
+ * WalletConnect payloads
+ */
+export const asCosmosWcGetAccountsRpcPayload = asObject({
+  method: asValue('cosmos_getAccounts'),
+  params: asObject({})
+})
+const asSignDirect = asValue('cosmos_signDirect')
```

Comment:
All the other ones use `asValue` directly in the object, but here you are doing it outside. Is there a reason for it to be different?